docs/controller.md

```diff
@@ -0,0 +1,93 @@
+## Controller - general information
+- Always up-to-date info with all the implementation details can be found in **src/controller.py**.
+- It is running on port 9001.
+- Requests must be in JSON structure, command are specified in `type` key, e.g. `{"type": "ping"}`.
+- All arguments (if needed) are being specified in the same root JSON, e.g. `{"type": "emulator-start", "version": "2.4.0"}`.
+- It is beneficial to always include `id` integer in the request, as it is being sent back in the response, and these two can be matched.
+- Response is always in JSON format, includes boolean `success` key, indicating if everything went as expected, e.g. `{"success": False, "error": "Unknown command - xyz"}`.
+- If the command does not ask anything, but simply wants to do something, the response will contain a confirmation in `response` key, e.g. `{"success": True, "response: "Emulator 2.4.0. started"}`.
+
+## Controller - supported commands
+
+- **ping**
+  - **response**: `{"response": "pong"}`
+
+- **background-check**
+  - **action**: check current status of bridge and emulator
+  - **response**: `{"bridge_status": bool, "emulator_status": bool}`
+
+- **emulator-start**
+  - **action**: start the specified version of emulator (and if one already runs, kills it)
+  - **arguments**:
+    - **version**: `str` (1.9.4, 2.4.0., etc.) - default is the latest TT
+    - **wipe**: `bool` ... whether to delete the emulator profile before starting it - default is False
+
+- **emulator-stop**
+  - **action**: stop the emulator
+
+- **emulator-setup**
+  - **action**: perform the emulator setup
+  - **arguments**:
+    - all appropriate to the `load_device()` function in `trezorlib`
+    - **mnemonic**: `str`
+    - **pin**: `str`
+    - **passphrase_protection**: `bool`
+    - **label**: `str`
+    - **needs_backup**: `bool` - default is False
+
+- **emulator-press-yes**
+  - **action**: press yes button on emulator
+
+- **emulator-press-no**
+  - **action**: press no button on emulator
+
+- **emulator-input**
+  - **action**: input a string into the emulator
```

Comment:
This might be confusing. I would say "Enter a string into a field on the emulator (such as passphrase entry)"